File: data/gpt-4o/readable_format_HPEC/cpp/file145.cpp
#include <stdio.h>
#include <omp.h>
int main()
{
 int numThreads = 0;
#pragma omp parallel
 {
  if (omp_get_thread_num() == 0) {
    numThreads = omp_get_num_threads();
  }
  else {
    printf("numThreads = %d
", numThreads);
  }
 }
 return 0;
}

File: data/StarCoder2/readable_format_HPEC/fortran/file145.f90
program DRB075_getthreadnum_orig_yes
    use omp_lib
    implicit none

    integer :: numThreads
    numThreads = 0

    !$omp parallel
    if ( omp_get_thread_num()==0 ) then
        numThreads = omp_get_num_threads();
    else
        print*,'numThreads =',numThreads
    end if
    !$omp endparallel
end program
